src/styles.ts

```diff
@@ -40,10 +40,9 @@ export const styles = css`
 		padding-top: var(--paper-input-container_-_padding-top, 8px);
 		padding-bottom: var(--paper-input-container_-_padding-bottom, 8px);
 		position: relative;
-
-		font-family: var(--font-family);
```

Comment:
Add the removed ones back. I don't think the inputs look the same without that ?

Reply:
They actually don't, I reverted this